client/blocks/app-banner/index.jsx

```diff
@@ -190,9 +238,36 @@ export class AppBanner extends Component {
 				</Card>
 			</div>
 		);
+	};
+
+	render() {
+		if ( ! this.props.shouldDisplayAppBanner || this.state.isDraftPostModalShown ) {
+			return null;
+		}
+
+		const displayJetpackAppBranding = config.isEnabled( 'jetpack/app-branding' );
+
+		return displayJetpackAppBranding
+			? this.getJetpackAppBanner( this.props )
+			: this.getWordpressAppBanner( this.props );
 	}
 }
 
+function BannerIcon( { icon } ) {
+	useEffect( () => {
+		const animation = lottie.loadAnimation( {
```

Comment:
Just learning about Lottie and I had a question: does the library take into account [`@media/prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) setting out of the box or would that need extra attention here?

Reply:
My understanding is that, no, [Lottie does not take into account motion preferences](https://github.com/airbnb/lottie-web/issues/1986#issuecomment-586582516). 

You raise a good point. It's an opportunity to improve this new feature. I would imagine we should render the last frame of the animation when reduced motion is preferred.

Reply:
Thanks for catching this! I'll work on a PR to take reduced animation preferences into account. 🙇‍♀️

Reply:
To tie up all the loose ends, I've gone ahead to create that PR here: https://github.com/Automattic/wp-calypso/pull/67826